package.json

```diff
@@ -59,7 +59,7 @@
     "requirejs": "2.1.18",
     "sinon": "1.16.1",
     "tslint": "4.5.1",
-    "typescript": "2.0.6",
+    "typescript": "2.1.5",
```

Comment:
2.2 is new and the typings aren't backwards compatible and some customers are still on 2.1